homeassistant/components/homekit/strings.json

```diff
@@ -8,18 +8,26 @@
             },
             "init": {
                 "data": {
-                    "include_domains": "[%key:component::homekit::config::step::user::data::include_domains%]"
+                    "include_domains": "[%key:component::homekit::config::step::user::data::include_domains%]",
+                    "filter_by_excluding": "Include entities not specifically excluded"
                 },
-                "description": "Entities in the \u201cDomains to include\u201d will be bridged to HomeKit. You will be able to select which entities to exclude from this list on the next screen.",
+                "description": "Entities in the \u201cDomains to include\u201d will be bridged to HomeKit. You will be able to select which entities are bridged from this list on the next screen.",
                 "title": "Select domains to bridge."
             },
             "exclude": {
                 "data": {
                     "exclude_entities": "Entities to exclude"
                 },
-                "description": "Choose the entities that you do NOT want to be bridged.",
+                "description": "Choose the entities that you do NOT want to be bridged. New entities will be automatically included.",
                 "title": "Exclude entities in selected domains from bridge"
             },
+            "include": {
+                "data": {
+                    "include_entities": "Entities to include"
+                },
+                "description": "Choose the entities that you DO want to be bridged. New entities will not be automatically included.",
```

Comment:
```suggestion
                "description": "Choose the entities that you want to be bridged. New entities will not be automatically included the next time the bridge is reloaded.",
```

Reply:
I'm not sure that's clearly articulating what happens. Homekit filtering is confusing already. Bridge restarting has nothing to do with this filter - it's essentially a "fixed" filter vs "exclude" which is more dynamic.

Reply:
I could see that it way.  I think the case above is really the import one here.

Reply:
OK, I'll work through these.... where did you land on text for this part that you'll accept?

Reply:
I think this one is fine, its the exclude that I'm concerned with as it will likely result in issues being opened about why entities aren't appearing automatically when they are added without the qualifer.